modules/meta.analysis/DESCRIPTION

```diff
@@ -25,21 +25,25 @@ Description: The Predictive Ecosystem Carbon Analyzer (PEcAn) is a scientific
     streamline the interaction between data and models, and to improve the
     efficacy of scientific investigation. The PEcAn.MA package contains
     the functions used in the Bayesian meta-analysis of trait data.
-Depends:
+Imports:
+    coda (>= 0.18),
     XML,
     lattice,
-    MASS,
     PEcAn.utils,
-    PEcAn.DB
-Imports:
-    coda (>= 0.18),
+    PEcAn.DB,
     PEcAn.logger,
+    MASS,
     PEcAn.settings,
     rjags
 Suggests:
+    ggmcmc,
+    rmarkdown,
+    knitr,
+    ggplot2,
```

Comment:
Purely aesthetic: Keeping dependency lists in alphabetical order helps when humans are reading them (the R checks don't care about the order).

```suggestion
    ggmcmc,
    ggplot2,
    knitr,
    rmarkdown,
```